testing/src/test/java/io/stargate/it/http/graphfirst/FilesResourceTest.java

```diff
@@ -49,4 +67,85 @@ public void shouldGetGraphQLDirectivesFile() throws IOException {
     assertThat(responseBody).isNotEmpty();
     assertThat(responseBody).contains("directive @cql_input");
   }
+
+  @Test
+  public void shouldGetFileWithSchema(@TestKeyspace CqlIdentifier keyspaceId) throws IOException {
+    // when get schema file
+    String url = String.format("%s:8080/graphqlv2/files/namespace/%s.graphql", host, keyspaceId);
```

Comment:
Can you use `ResourcePaths` instead of hard-coding `/graphqlv2/files`?
Paths are likely to change in the near future.

Reply:
the `testing` module does not have a dependency on the `graphql` module, therefore the `ResourcePaths` is not available.

Reply:
Good point. Maybe we can extract the paths in the parent test class then. They'll still be duplicated, but only in one place.